assets/package.json

```diff
@@ -8,9 +8,11 @@
     "build:dev": "webpack --mode development"
   },
   "dependencies": {
+    "bootstrap": "^4.6.0",
```

Comment:
Prefer to use exact versions to enforce stable builds. Even patch releases of JS dependencies can break a build and introduce differences between environments.

Reply:
Noted & fixed in b085d4e.
> Ideally, the renaming of the project and set up of Bootstrap should be have been done in separate PRs to avoid having a PR with 80+ file changes

also noted to avoid it next time 🙈